framework/caching/Cache.php

```diff
@@ -462,7 +474,9 @@ protected function getValues($keys)
     {
         $results = [];
         foreach ($keys as $key) {
+            $this->beginProfile($key, __METHOD__);
```

Comment:
Sorry I did not understand the question first. Maybe a combination of the keys via implode() that could get really long however.